openapi/connector_mgmt.yaml

```diff
@@ -1122,9 +1117,8 @@ components:
           properties:
             collections:
               type: array
-              items:
-                allOf:
-                  - $ref: "#/components/schemas/ObjectReference"
+              items: # removed allOf to workaround this issue: https://github.com/microsoft/kiota/issues/2442
+                $ref: "#/components/schemas/ObjectReference"
```

Comment:
this is the workaround I'm sneaking in, the generated code is not affected and the signature is pretty much compatible with the previous; I do believe that it's an acceptable change to make the life of the SDKs generator easier 🙂